### PR TITLE
Clean Up Repo

### PR DESCRIPTION
**Pull request by kiawnna**
@nick-bair @RickyAmparoCloud

### CODE REVIEW

File: AddRecordsSeedScript/addRecordsSeedScript.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+headers = {
+    "Authorization": f'User {os.getenv("USER_SECRET")}, Organization {os.getenv("ORGANIZATION_SECRET")}, Element {os.getenv("INSTANCE_TOKEN")}',
+    "Content-Type": "application/json",
+    "accept": "application/json"
+}
+
+url = f'https://staging.cloud-elements.com/elements/api-v2/{object}'
+
+try:
+    for i in range(0, 3):
```
Comment (nick-bair) [inline AddRecordsSeedScript/addRecordsSeedScript.py:47]: @kiawnna  Instead of range(0, 3), should it be range(0, max_records)?

**Review by nick-bair — CHANGES_REQUESTED** [review]
@kiawnna EPIC!  Only one tiny request for changes.

File: AddRecordsSeedScript/addRecordsSeedScript.py (hunk 1 above)
Comment (kiawnna) [reply AddRecordsSeedScript/addRecordsSeedScript.py:47]: yup!

**Comment by kiawnna** [comment]
@nick-bair Done! :)

**Review by nick-bair — APPROVED** [review]

**Pull request merged**